static/main.css

```diff
@@ -1,144 +1,146 @@
 @import url("https://fonts.googleapis.com/css?family=Montserrat&display=swap");
 
 body {
-  font-family: "Montserrat", sans-serif;
-  font-size: 14px;
-  margin: 20px 10px 10px 10px;
-  padding: 0;
-  border: 0;
+	border: 0;
+	font-family: "Montserrat", sans-serif;
+	font-size: 14px;
+	margin: 20px 10px 10px 10px;
+	padding: 0;
 }
 
 button:hover {
-  cursor: pointer;
+	cursor: pointer;
 }
 
 button:focus {
-  outline: none;
+	outline: none;
 }
 
 input {
-  border-radius: 4px;
-  border: 1px solid #bbb;
-  outline: none;
+	border-radius: 4px;
+	border: 1px solid #bbb;
+	outline: none;
 }
 
 .columns {
-  border-radius: 5px;
-  display: flex;
-  flex-wrap: wrap;
-  align-items: flex-start;
-  background-color: #ececec;
-  padding: 5px 0 5px 10px;
-  margin-top: 10px;
+	align-items: flex-start;
+	background-color: #ececec;
+	border-radius: 5px;
+	display: flex;
+	flex-wrap: nowrap;
+	margin-top: 10px;
+	overflow-x: auto;
+	padding: 5px 0 5px 10px;
 }
 
 .columns .column {
-  background: #99eaa4;
-  border-radius: 5px;
-  margin: 5px 10px 5px 0;
-  padding: 0 5px 5px 5px;
-  display: flex;
-  width: 400px;
-  position: relative;
-  justify-content: center;
-  align-items: center;
-  flex-direction: column;
+	align-items: center;
+	background: #99eaa4;
+	border-radius: 5px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin: 5px 10px 5px 0;
+	max-width: 400px;
+	min-width: 400px;
+	padding: 0 5px 5px 5px;
+	position: relative;
```

Comment:
Sorry, I reordered attributes. Changes were setting flex-wrap on columns to nowrap and adding overflow-x: auto;

Added max/min width to column class